### DISCUSSION TITLE
Does it support custom chart dynamic titles？支持自定义图表动态标题吗

### DISCUSSION THREAD

**Discussion by aimmon** [apache/superset#25999] [Q&A / Help] [answered]
Superset TAG:3.0.0rc4

I am using the latest version of Superset. I want to set the title of the chart and use the value of a certain control in the filter. Does Superset support this feature
我用的是最新版本的Superset，我想设置图表的标题，用筛选器的某个控件的值，Superset支持这个功能吗
Alternatively, using the text and header implementation of layout elements is also possible
或者用layout elements 的text , header 实现，也是可以的
Thank you.
![or](https://github.com/apache/superset/assets/20752671/30e1a558-197b-47d5-909d-1ad43c40f518)

>

**Comment by sfirke** [comment, marked as answer]
No, unfortunately Superset does not currently support dynamic chart titles.

**Reply by aimmon** [reply]
Thank you. I hope that future versions can consider implementing this possibility

**Comment by PutingZhao** [comment]
mark
